modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/utils/sectionAddedHandler.js

```diff
@@ -110,7 +110,11 @@ const handleSectionAdded = (props, state, event) => {
 	else if (existingField.type === FIELD_TYPE_FIELDSET) {
 		newPages = addFieldToPage({
 			...props,
-			indexes,
+			indexes: {
```

Comment:
Have you talked to Luiz about putting the new fields on top of all fields of the fields group in this scenario? I'm only asking you this, because, when you create a fields group by dragging a new field to an existing one, the new field is placed below the existing one in the newly created group. Also, adding new fields to a form by double clicking the sidebar puts them at the bottom of the page as well.

Reply:
Good point! I'll talk to him.

Reply:
I talked with Julia and Luiz and they both agreed that the field should be inserted at the bottom, as you said.
I've updated the code with this change.

Thanks a lot, @rodrigopaulino!